### DISCUSSION TITLE
Validate Console ID 100 (Hubs)

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#806] [Ideas]
Would like to have console ID 100 (Hubs) validated, similar to 101 (Events).

I've wanted to work on obtainable badges for hubs since they became a thing and lately there's more and more people speculating about it. 

Obviously the whole thing would need to be approved first, but I can't work on a prototype to pitch it if I can't upload achievements to a hub entry.

**Comment by HolyShinx** [comment]
what types of hubs would deserve a badge, also how would it be possible to stop people from making hubs so they get a free badge? I'm really interested in that topic

**Comment by televandalist** [comment]
I'm not worried about people making them cause it would work similar to events. Generally not suppose to have hashes linked, so nothing can be earned without a manual unlock. If you do, then you are very naughty and get an untrack. :) It's worked for events. Technically anyone with dev perms could add to event entries, they just don't. 

Anyway, the hubs would probably need to meet a handful of criteria before actually getting a badge. 

A hub that gets added to over time would probably not work. Should be stuff that won't change. I think the easiest example is for classic Mega Man. There's only ever going to be Mega Man 1-6 (NES) in that category.

**Comment by HolyShinx** [comment]
wow it sounds so good, can't wait

**Comment by Blazekickn** [comment]
> A hub that gets added to over time would probably not work. Should be stuff that won't change.

I mean i would like to see badges for stuff like the hacks hubs, even if they do get more things added to them over time. Doing all the smw hacks alone means you have do 100% like 50+ games which definitely deserves something
